sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/entries/CreateNewTravelEntrySteps.java

```diff
@@ -226,6 +259,58 @@ public CreateNewTravelEntrySteps(
               "User name is invalid");
           softly.assertAll();
         });
+
+    When(
+        "^I check if pick or create person popup in travel entry is displayed$",
+        () -> {
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(
+              PICK_OR_CREATE_PERSON_TITLE_DE); // wait for popup
+          String expectedTitle = "Person ausw\u00E4hlen oder erstellen";
+          String checkPopupTitle =
+              webDriverHelpers
+                  .getTextFromWebElement(PICK_OR_CREATE_PERSON_TITLE_DE)
+                  .toLowerCase(Locale.GERMAN);
+          softly.assertEquals(
+              checkPopupTitle,
+              expectedTitle.toLowerCase(Locale.GERMAN),
+              "Wrong popup title for Pick or create person");
+          softly.assertAll();
+        });
+
+    When(
+        "^I check Pick an existing case in Pick or create person popup in travel entry$",
+        () -> webDriverHelpers.clickOnWebElementBySelector(PICK_A_EXISTING_PERSON_LABEL_DE));
+
+    When(
+        "^I click confirm button in popup from travel entry$",
+        () -> webDriverHelpers.clickOnWebElementBySelector(SAVE_POPUP_CONTENT));
+
+    When(
+        "I choose an existing case while creating case from travel entry",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(
+              PICK_A_EXISTING_CASE_LABEL_DE); // wait for popup
+          String expectedTitle = "Fall ausw\u00E4hlen oder erstellen";
+          String checkPopupTitle =
+              webDriverHelpers
+                  .getTextFromWebElement(PICK_OR_CREATE_PERSON_TITLE_DE)
+                  .toLowerCase(Locale.GERMAN);
+          softly.assertEquals(
+              checkPopupTitle,
+              expectedTitle.toLowerCase(Locale.GERMAN),
+              "Wrong popup title for Pick or create a case");
+          softly.assertAll();
+          webDriverHelpers.clickOnWebElementBySelector(PICK_A_EXISTING_CASE_LABEL_DE);
+        });
```

Comment:
same, method name is general, but code logic covers DE. Can we make a generic method to cover both en and de?

Reply:
we still have no scenario for en version, if will be, i don't know who will be do that and what kind of idea will have for that, in my opinion if somebody wanna reuse it then, no problem to change by him and make directly with need